manifests/4.5/cluster-loggings.crd.yaml

```diff
@@ -19,6 +19,14 @@ spec:
           description: Specification of the desired behavior of the Logging cluster.
           type: object
           properties:
+            metadata:
+              description: Specification of the metadata for the cluster
+              properties:
+                name:
+                  description: The name of the clusterlogging object
+                  enum:
+                    - instance
```

Comment:
nit -- not sure if this will affect anything but we may want to un-indent this to match formatting (not sure if `go fmt` would catch this)